ui/js/controllers/perf/compare.js

```diff
@@ -837,3 +837,34 @@ perf.controller('CompareSubtestResultsCtrl', [
             });
         });
     }]);
+
+perf.controller('toolTipGraphsCtrl', [
+    function toolTipGraphsCtrl() {
+        var a_canvas = document.getElementById("a");
+        var a_context = a_canvas.getContext("2d");
+        //var data1 = [1, 2, 3, 4, 5];
+        //var data1 = [435.96, 435.91, 443.05, 584.36, 585.14, 585.55, 587.12, 588.38, 588.69, 589.25, 590.73];
+        var data1 = [37, 44, 71, 19, 25, 66, 91, 83, 52, 26];
+        var max = _.max(data1, function (element) {return element;})*1.25;
+        var min = _.min(data1, function (element) {return element;})/1.25;
+        //draw borders
+        a_context.strokeRect(0, 0, 400, 30);
+        a_context.fillStyle = '#222222';
+        a_context.fillRect(0, 0, 50 ,30);
```

Comment:
There are some cases where we would likely have large numbers here (i.e. the memory tests: https://treeherder.mozilla.org/perf.html#/graphs?series=%5Bmozilla-inbound,a630e0d4f7000610ca57d0f8da52b55d117632a9,1,4%5D)

I think we need to format large numbers into some kind of smaller string (i.e. 1260000 -> 1.26m). The numeraljs library looks useful here: http://numeraljs.com/

I think we should be able to just import it into our javascript by add it to vendor/ and importing it inside `ui/perf.html`